en/docs/learn/outbound-provisioning-with-salesforce.md

```diff
@@ -234,7 +234,7 @@ Provider.
     information on registering and configuring an identity provider.
 4.  Upload the Salesforce public certificate that you generated and
     saved in [step 7 under Configuring
-    Salesforce](outbound-provisioning-with-salesforce#public)
+    Salesforce](../../learn/outbound-provisioning-with-salesforce)
```

Comment:
this link path is missing the `#anchor`
it should be `../../learn/outbound-provisioning-with-salesforce#public`
and the anchor tag should be added to the relevant place.

please fix all relevant places